dashboard/README.md

```diff
@@ -1,7 +1,23 @@
 # Dashboard!!
```

Comment:
Double exclamation marks,  unprofessional.

Reply:
Pedro can format this as needed.